apisix/core/config_etcd.lua

```diff
@@ -133,6 +133,9 @@ end
 
 
 local function sync_data(self)
+    math.randomseed(tostring(os.time()))
```

Comment:
useless, the apisix has set the `randomseed` in `init` phase

Reply:
Thank you. Used for test. Forget to remove my bad